src/sentry/search/events/fields.py

```diff
@@ -1314,6 +1314,24 @@ def with_default(default, argument):
     return argument
 
 
+class SnQLNumericColumnNoLookup(NumericColumn):
+    measurement_aliases = {
```

Comment:
Doesn't seem like this is being used anymore

Reply:
removed!